connect/src/main/java/com/stripe/android/connect/analytics/ConnectAnalyticsEvent.kt

```diff
@@ -31,7 +31,7 @@ internal sealed class ConnectAnalyticsEvent(
      * Note: This should happen before component_loaded, so we won't yet have a page_view_id.
      */
     data class WebPageLoaded(
-        val timeToLoad: Double
+        val timeToLoad: Long
```

Comment:
Document the unit (ms, seconds)?

Reply:
On iOS, this is a `Double` and the unit is in seconds:
https://docs.google.com/document/d/1nzHGofoclzij4nP5n5qgSpYLia-93heStXvt7vOfrkQ/edit?tab=t.0#bookmark=id.1au4f34vo3wd

Was this changed to `Long` to be compatible with the Android `Clock` API? Maybe we could store it milliseconds using `Long` and then convert it inside `mapOf` to keep it easy to construct.